modules/appeals_api/bin/rails

```diff
@@ -2,7 +2,7 @@
 # frozen_string_literal: true
 
 ENGINE_ROOT = File.expand_path('..', __dir__)
-ENGINE_PATH = File.expand_path('../lib/vba_documents/engine', __dir__)
```

Comment:
Found this while working on this ticket. Seems incorrect. Please see my change and let me know.